### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/EAST/commit/597d28110b790d5fbbc0afc4b47c36cc14566143#commitcomment-54654074

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/EAST/commit/07dd3cb93cbbfb681b7bd80cd82cd09bce7381d6

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: fake_mail_server/tests/create_configs.rs [hunk 1]
```diff
@@ -205,7 +205,7 @@ fn ok_greeting_codes() {
     ];
     for cap_combo in combos(&caps) {
         let mut cap_str = String::new();
-        cap_str.push_str("CAPABALITY");
+        cap_str.push_str("CAPABILITY");
```
Comment (jsoref) [inline fake_mail_server/tests/create_configs.rs:208]: This one intrigues me

File: scanning_scripts/modules/imap_implicit_tls/scanner.go [hunk 2]
```diff
@@ -14,15 +14,15 @@ type ScanResults struct {
 	Greeting string `json:"greeting,omitempty"`
 
 	// Capabilities is the targets response to CAPA
-	Capabilties string `json:"Capabilties,omitempty"`
+	Capabilities string `json:"Capabilities,omitempty"`
```
Comment (jsoref) [inline scanning_scripts/modules/imap_implicit_tls/scanner.go:17]: I don't think this matters

File: scanning_scripts/modules/pop3_implicit_tls/scanner.go [hunk 3]
```diff
@@ -14,12 +14,12 @@ type ScanResults struct {
 	Greeting string `json:"help,omitempty"`
 
 	// Capabilities is the targets response to CAPA
-	Capabilties string `json:"Capabilties,omitempty"`
+	Capabilities string `json:"Capabilities,omitempty"`
```
Comment (jsoref) [inline scanning_scripts/modules/pop3_implicit_tls/scanner.go:17]: I don't think this matters

File: scanning_scripts/modules/smtp_capabilities/scanner.go [hunk 4]
```diff
@@ -26,7 +26,7 @@ type ScanResults struct {
 	// PostTLS is the targets response to EHLO in encrypted (post-tls) state
 	PostTLS string `json:"postTLS,omitempty"`
 
-	// Help is the targets response to the HELP command in encrypte (tls) state
+	// Help is the targets response to the HELP command in encrypted (tls) state
```
Comment (jsoref) [inline scanning_scripts/modules/smtp_capabilities/scanner.go:29]: I think this is correct (it's halfway between `encrypt` and `encrypted`).

Note that `targets` should be `target's`, but this is a spell checker, not a grammar check....

File: fake_mail_server/tests/create_configs.rs (hunk 1 above)
Comment (Murgeye) [reply fake_mail_server/tests/create_configs.rs:208]: This is (somewhat) old code that we originally wanted to use for generating test cases. This never really went in to use (but having this typo would have made for a fun debugging session 😄 )

**Review by Murgeye — APPROVED** [review]
Thanks!

**Pull request merged**